prover/crates/bin/prover_autoscaler/src/global/scaler.rs

```diff
@@ -48,6 +49,16 @@ static PROVER_DEPLOYMENT_RE: Lazy<Regex> =
 static PROVER_POD_RE: Lazy<Regex> =
     Lazy::new(|| Regex::new(r"^circuit-prover-gpu(-(?<gpu>[ltvpa]\d+))?").unwrap());
 
+/// gpu_to_prover converts Gpu type to corresponding deployment name.
+fn gpu_to_prover(gpu: Gpu) -> String {
+    let s = "circuit-prover-gpu";
+    match gpu {
+        Gpu::Unknown => "".into(),
+        Gpu::L4 => s.into(),
+        _ => format!("{}-{}", s, gpu.to_string().to_lowercase()),
```

Comment:
T4 and others are covered by default section: for all GPUs except of L4 we named deployment "circuit-prover-<gpu>", and only for L4 it's just "circuit-prover".

I think it would be better to rename deployments while we still can...